WooCommerce/WooCommerceScreenshots/Screens/Orders/OrderSearchScreen.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import XCTest
+
+class OrderSearchScreen: BaseScreen {
+
+    //Note – search field is unreliable on non-iOS 13 devices
```

Comment:
Is this a bug? If so, let's make sure it's reported.

Reply:
This comment wasn't clear – iOS 13 has an alternate code path for `UISearchBar` in the Woo code. I adjusted this to a (slightly) worse, but more common configuration. We can't target the searchField using `XCUIApplication().searchFields`, but by using `XCUIApplication().otherElements`, we now have a common code path for tests, which is better.